src/components/inputs/Input.tsx

```diff
@@ -71,7 +71,7 @@ const Input = <T extends InputValue>({
   const showCustomValueRendering = typeof renderedValue !== 'string' && renderedValue !== undefined
 
   const labelStyle = useAnimatedStyle(() => ({
-    top: withSpring(!isActive ? -10 : -40, fastSpringConfiguration)
+    bottom: withSpring(!isActive ? 0 : 30, fastSpringConfiguration)
```

Comment:
Good catch :)

Reply:
a frustrating one! Thanks to the inspect mode